### PR TITLE
Updated the vscode whitespace settings.

### PR DESCRIPTION
**Pull request by usergenic**
We need these settings everywhere.  Grabbed from `polymer-analyzer`.

### CODE REVIEW

**Comment by usergenic** [comment]
@rictic suddenly type errors showing up in travis so I added some `!` to a test.

**Comment by usergenic** [comment]
https://travis-ci.org/Polymer/polymer-bundler/builds/220755209 for reference.

File: src/test/bundler_test.ts [hunk 1]
```diff
@@ -196,9 +196,9 @@ suite('Bundler', () => {
     const doc = await bundle('test/html/import-in-body.html');
     const imports = dom5.queryAll(doc, importMatcher);
     assert.equal(imports.length, 0);
-    const bodyContainer = dom5.query(doc, bodyContainerMatcher);
-    const scriptActual = dom5.query(doc, scriptExpected)!.parentNode;
-    const divActual = dom5.query(doc, divExpected)!.parentNode;
+    const bodyContainer = dom5.query(doc, bodyContainerMatcher)!;
```
Comment (rictic) [inline src/test/bundler_test.ts]: This is the weirdest one. assert.equal requires non-null values?

Comment (usergenic) [inline src/test/bundler_test.ts]: Yeah this just started.   Now anything cut from master is broken without the fix.

Comment (usergenic) [inline src/test/bundler_test.ts]: Gracias!

**Pull request merged**